internal/analytics/events.go

```diff
@@ -42,8 +45,36 @@ func MockedTracker(name string, action string, flags []string) *MockTracker {
 			"id":      id,
 			"name":    name,
 		},
-	}
-	tracker := MockTracker{ID: id}
-	tracker.On("SendEvent", mockedTrackingArgs...)
+	}...)
+	tracker.On("SendEvent", []interface{}{
+		"testAccessToken",
+		"http://test.com",
+		"CLI Command Completed",
+		map[string]interface{}{
+			"id":      id,
+			"outcome": outcome,
+		},
+	}...)
 	return &tracker
 }
+
+func SendCommandRunEvent(name string, cmd *cobra.Command, analyticsTracker Tracker) {
+	cmd.Root().Annotations = map[string]string{"tracking": "true"}
+	analyticsTracker.SendEvent(
+		viper.GetString(cliflags.AccessTokenFlag),
```

Comment:
I mentioned this in another PR, but the internal package shouldn't reference framework code like cobra or viper. We should pass in these values instead of reading them from viper. It also means we should move `CmdRunEventProperties` to the `cmd` package (or a subpackage) -- maybe `cmd/analytics`?
* `internal/analytics` - "domain model" for analytics tracking
* `cmd/analytics` - translation layer between framework code and domain code